package.json

```diff
@@ -38,7 +38,8 @@
 		"e2e:dev": "NODE_ENV=development playwright test",
 		"e2e:ci": "playwright test --reporter=html",
 		"e2e:snapshots": "npx playwright test --update-snapshots --reporter=list",
-		"e2e:report": "npx playwright show-report"
+		"e2e:report": "npx playwright show-report",
+		"erc20:check": "npx vite-node scripts/check.tokens.erc20.ts"
```

Comment:
Shouldn't it be `npm run vite-node` or just `vite-node` given that the dependencies is installed?

Reply:
you are right! I forgot to change it there...